src/components/ShareExperience/InterviewForm/InterviewExperience/InterviewSensitiveQuestions.js

```diff
@@ -80,7 +80,10 @@ const InterviewSensitiveQuestions = ({
             value={otherValue || ''}
             onChange={e => onChange([...resultsBesidesOther, e.target.value])}
             placeholder="輸入敏感問題"
-            warningWording="請輸入20個字以內"
+            warningWording="請輸入 1～20 字"
+            isWarning={
+              !otherValue || otherValue.length > 20 || otherValue.length === 0
+            }
```

Comment:
需要將submitted 考慮進去，從上層可以直接傳入。大致如下：
```suggestion
  isWarning={
             submitted && ( !otherValue || otherValue.length > 20 || otherValue.length === 0)
  }
```
然後如果方便，可以幫忙把整個邏輯抽在return 之前，比較清晰一點，感謝。